pkg/sloop/store/typed/watchtablegen.go

```diff
@@ -190,7 +190,7 @@ func (t *KubeWatchResultTable) getLastMatchingKeyInPartition(txn badgerwrap.Txn,
 	itr.Seek([]byte(keySeekStr))
 
 	// if the result is same as key, we want to check its previous one
-	if oldKey == string(itr.Item().Key()) {
+	if itr.Valid() && oldKey == string(itr.Item().Key()) {
```

Comment:
this should go to `tabletemplate.go`, otherwise it will be overridden next time we run go gen

Reply:
will open a new pr for that change, 1. this pr only focus on payload changes which only touch watch table. 2. make the pr short for reviewing purpose.